Example/src/LightboxExample.js

```diff
@@ -121,7 +121,7 @@ const timingConfig = {
   easing: Easing.bezier(0.33, 0.01, 0, 1),
 };
 
-const HEADER_HEIGHT = Header.HEIGHT - StatusBar.currentHeight;
+let HEADER_HEIGHT = StatusBar.currentHeight;
```

Comment:
This shouldn't be uppercased if it's not a constant. Couldn't it be moved to components?